include/vas.h

```diff
@@ -90,6 +90,15 @@ do {									\
 	VAS_Fail(__func__, __FILE__, __LINE__, expl, VAS_WRONG);	\
 } while (0)
 
+#define PTOK(call)							\
+do { 									\
+	int _pterr = call;						\
```

Comment:
could use `VUNIQ_NAME()`, but that's not too relevant

Reply:
I completely forgot about this macro, we could use it in `REPLACE()` too.

Reply:
lgtm